core/domain/stats_services_test.py

```diff
@@ -2350,6 +2350,63 @@ def test_update_learner_answer_details(self):
         self.assertEqual(
             len(learner_answer_details.learner_answer_info_list), 2)
 
+    def test_get_learner_answer_info_for_exploration(self):
+        owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        user_services.create_new_user(owner_id, self.OWNER_EMAIL)
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.save_new_valid_exploration(self.exp_id, owner_id)
+        exploration = exp_fetchers.get_exploration_by_id(self.exp_id)
```

Comment:
nit: please use `exp` to match with `exp_id`.

Reply:
Done